packages/console/src/pages/Connectors/components/CreateForm/index.tsx

```diff
@@ -56,7 +57,14 @@ const CreateForm = ({ onClose, isOpen: isFormOpen, type }: Props) => {
             existingConnectors.some(({ connectorId }) => connector.id === connectorId),
         })),
       }))
-      .filter(({ connectors }) => !connectors.every(({ added }) => added));
+      .filter(({ connectors }) => !connectors.every(({ added }) => added))
+      .slice()
+      .sort((connectorA, connectorB) => {
+        const orderA = getConnectorOrder(connectorA.target, connectorA.isStandard);
+        const orderB = getConnectorOrder(connectorB.target, connectorB.isStandard);
+
+        return orderA - orderB;
```

Comment:
nit: the result of `Number.POSITIVE_INFINITY - Number.POSITIVE_INFINITY` is `NaN`, will that affect the order result?

Reply:
updated to a safer value.